Changes for RHDEVDOCS-4829 Doc: ODC Usability & Customer Happiness Epic (#55909)
openshift/openshift-docs: applications/working_with_helm_charts/understanding-helm.adoc

Helm uses a packaging format called _charts_.
A Helm chart is a collection of files that describes the {product-title} resources.

> A running instance of the chart in a cluster is called a _release_. A new release is created every time a chart is created on the cluster.

Reviewer: For minimalism, maybe:
```suggestion
Creating a chart in a cluster creates a running instance of the chart known as a _release_.
```

Author: Updated, thanks!
